LESS/index.less

```diff
@@ -0,0 +1,14 @@
+// Variables and Mixins
+@import "./variables.less";
+@import "./mixins.less";
+
+// Reset and Global styles
+@import "./reset.less";
+@import "./global.less";
+
+// Reusable Components
+@import "./navigation.less";
+@import "./footer.less";
+
+// Page Specific
+@import "./home.less";
```

Comment:
Just a note, not a big deal, matter of pref but in the future if you want to leave of the .less extension you can - the compiler will assume that is the file if presented with no extension.